### PR TITLE
add missing inject

### PR DESCRIPTION
**Pull request by felipeweb**

### CODE REVIEW

File: vraptor-core/src/main/java/br/com/caelum/vraptor/serialization/I18nMessageSerialization.java [hunk 1]
```diff
@@ -33,30 +33,31 @@
...
+    private I18nMessage i18nMessage;
+
+    /**
+     * @deprecated CDI eyes only
+     */
+    protected I18nMessageSerialization() {
+        this(null, null);
+    }
+
+    @Inject
```
Comment (Turini) [inline vraptor-core/src/main/java/br/com/caelum/vraptor/serialization/I18nMessageSerialization.java]: ops, nice catch @felipeweb!

**Comment by Turini** [comment]
amazing, thanks for sending this PR, @felipeweb! 
Could you just fix the indentation to use tabs instead of spaces?

**Comment by felipeweb** [comment]
done!! sorry about my indent.

**Comment by Turini** [comment]
great! thank you again, Felipe!

**Pull request merged**